src/main/kotlin/thatline/localup/etcapi/service/WeatherService.kt

```
@@ -0,0 +1,149 @@
+    fun getThreeDayWeatherSummaries(
+        sigunguCode: String,
+    ): WeatherInformation {
+        val baseDateTime = LocalDateTime.now()
+            .minusMinutes(10) // 단기예보조회, API 제공 시간 10분 보정
+            .truncatedTo(ChronoUnit.HOURS)
+        val baseDate = baseDateTime.format(DateTimeUtil.DATETIME_FORMATTER_yyyyMMdd)
+        val baseTime = "0200"
+
+        // TODO: 로직 개선 필요
+        val tourApiArea = TourApi.getTourApiAreaBySigunguCode(sigunguCode)
+            ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)